openassessment/xblock/static/js/src/oa_server.js

```diff
@@ -137,6 +137,29 @@ if (typeof OpenAssessment.Server === "undefined" || !OpenAssessment.Server) {
         },
 
         /**
+         * Renders the count of ungraded and checked out assessemtns.
+         *
+         * @returns {promise} A JQuery promise, which resolves with the HTML of the rendered section
+         *     fails with an error message.
+         */
+        staffGradeCounts: function() {
+            var url = this.url('render_staff_grade_counts');
+            return $.Deferred(function(defer) {
+                $.ajax({
+                    url: url,
+                    type: "POST",
+                    dataType: "html"
+                }).done(function(data) {
+                    defer.resolveWith(this, [data]);
+                }).fail(function() {
+                    defer.rejectWith(
+                        this, [gettext('The display of ungraded and checked out assessments could not be loaded.')]
```

Comment:
I think "assessments" -> "responses":
"The display of ungraded and checked out responses could not be loaded"